jpsonic-main/src/main/java/com/tesshu/jpsonic/service/MediaFileService.java

```diff
@@ -177,6 +177,8 @@ MediaFile checkLastModified(final MediaFile mediaFile, boolean useFastCache) {
                         && mediaFile.getChanged().getTime() >= FileUtil.lastModified(mediaFile.getFile())
                         && !ZERO_DATE.equals(mediaFile.getLastScanned())) {
                     return mediaFile;
+                } else if (settingsService.isIgnoreFileTimestamps() && !ZERO_DATE.equals(mediaFile.getLastScanned())) {
```

Comment:
@sonatype-lift ignore
Currently, the operation is guaranteed by testing. If you're replacing it with a new API, it should be exhaustive rather than sporadic.